packages/instant-meilisearch/tsconfig.test.json

```diff
@@ -2,7 +2,6 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "suppressImplicitAnyIndexErrors": true,
-    "resolveJsonModule": true
```

Comment:
👀 can you explain that 😅 ?

Reply:
It's already set as `true` in the `tsconfig.json` this config file extends:
https://github.com/meilisearch/meilisearch-js-plugins/blob/2bd616385ac42ab55746184cc92b3a83a8e8a2b8/packages/instant-meilisearch/tsconfig.json#L26

- https://www.typescriptlang.org/tsconfig#extends